lib/tasks/temp.rake

```diff
@@ -1,6 +1,14 @@
 # -*- coding: utf-8 -*-
 namespace :temp do
 
+  desc 'Populate last_event_time column of InfoRequest'
+  task :populate_last_event_time => :environment do
+    InfoRequest.where('last_event_time is NULL').find_each do |info_request|
```

Comment:
Super minor pedantry: `is` could be capitalised too if we're capitalising `NULL` in the SQL.